action/protocol/execution/protocol_test.go

```diff
@@ -403,6 +404,9 @@ func (sct *SmartContractTest) prepareBlockchain(
 	if sct.InitGenesis.IsLondon {
 		cfg.Genesis.Blockchain.ToBeEnabledBlockHeight = 0
 	}
+	if sct.IsLegacyNonceAccount {
+		cfg.Genesis.Blockchain.ToBeEnabledBlockHeight = 4
+	}
```

Comment:
let's use `sct.InitGenesis.IsLondon` and delete `sct.InitGenesis.IsLondon`

Reply:
delete sct.IsLegacyNonceAccount